### PR TITLE
Redocly Integration in docs

### PR DESCRIPTION
**Pull request by Freedisch**
**Notes for Reviewers**
fixed potential overlapping errors during integration
This PR fixes 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
DCO...

**Comment by l5io** [comment]
🚀 Preview for commit 2f5ae8eeffb5061ced50cd390b56dc7726592909 at: https://64a310f568b3942071bac1c9--meshery-docs-preview.netlify.app

File: docs/_layouts/redocly.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+    </style>
+    {{{redocHead}}}
+  </head>
+  <body>
+    {{{redocHTML}}}
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply docs/_layouts/redocly.html:17]: When redoc-static.html is produced from the new redocly-docs-build target, I don't see this HTML template being applied.

File: docs/redocly.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+extends:
+  - recommended
+rules:
+  no-unused-components: error
+theme:
+  openapi:
+    htmlTemplate: _layouts/redocly.html
+    sortTagsAlphabetically: true
+
+    # theme:
```
Comment (leecalcote) [inline docs/redocly.yaml:15]: @Freedisch is none of this configurable?

File: docs/_layouts/redocly.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (Freedisch) [reply docs/_layouts/redocly.html:17]: This html template is referenced in the `redocly.yml` and will be applied during the built

File: docs/redocly.yaml (hunk 2 above)
Comment (Freedisch) [reply docs/redocly.yaml:15]: No, It's not. By default, all these functionalities are not applicable

**Comment by Chadha93** [comment]
@Freedisch Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by l5io** [comment]
🚀 Preview for commit 96afae368442ad53cd972086c1274a5b9e70756a at: https://64a4717b55f474106f3681fb--meshery-docs-preview.netlify.app

**Comment by 864893031735973** [comment]
Usd

**Pull request closed without merging**